### DISCUSSION TITLE
comments

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#74] [Announcements]
# comments

想说什么就说点什么吧

https://blog.ops-coffee.cn/comments.html

**Comment by xiamuguizhi** [comment]
https://ops-coffee.cn/ 的导航效果真的很棒！

**Reply by ops-coffee** [reply]
我也觉得，哈哈

**Comment by SmallCarmelon** [comment]
看了老哥的象山游记，应该是个热爱生活的人，未来打算跟着笔记浙江省内走一圈

**Reply by ops-coffee** [reply]
那你可是找对攻略了，预祝顺利～

**Comment by xuanskyer** [comment]
博主这网站是用什么程序搭建的啊

**Reply by ops-coffee** [reply]
自己写的，细节可以看看这里：https://blog.ops-coffee.cn/life/writing.html

**Comment by 9201314** [comment]
对博主的旅行地图一见钟情，和博主一样差不多也游过了六七个省，也想有生之年走遍祖国的大江南北。想单独搭一个旅行地图的页面，只是用来展示各地的风景照，尝试保存博主的页面没折腾明白，能否把旅行地图单独开源出来😁

**Comment by hsiaohui** [comment]
https://tool.huei.cc/
抄了大佬的作业，感谢大佬开源😎
